content/en/docs/tasks/configure-pod-container/configure-gmsa.md

```diff
@@ -191,4 +196,10 @@ spec:
         beta.kubernetes.io/os: windows
 ```
 
+### Conclusion
```

Comment:
Following the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice), we should use an active voice. (ex: The mutating webhook will resolve -> The mutating webhook resolves... for all 3 examples)